mbot_gazebo:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
  
  # Head position Controllers ---------------------------------------
  head_angle_controller:
    type: effort_controllers/JointPositionController
    joint: mbot_head_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  
  # Wheel velocity Controllers ---------------------------------------
  back_wheel_controller:
    type: velocity_controllers/JointVelocityController
    joint: back_wheel_joint
#    pid: {p: 100.0, i: 0.01, d: 10.0}

  left_wheel_controller:
    type: velocity_controllers/JointVelocityController
    joint: left_wheel_joint
#    pid: {p: 100.0, i: 0.01, d: 10.0}

  right_wheel_controller:
    type: velocity_controllers/JointVelocityController
    joint: right_wheel_joint
#    pid: {p: 100.0, i: 0.01, d: 10.0}

  # PID gains ---------------------------------------
  gazebo_ros_control/pid_gains:
    back_wheel_joint:  {p: 100.0, i: 0.01, d: 10.0}
    left_wheel_joint:  {p: 100.0, i: 0.01, d: 10.0}
    right_wheel_joint: {p: 100.0, i: 0.01, d: 10.0}

# EOF
